vcpkg/patches/oss/0003-vcpkg-dependencies-sanitizer.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/scripts/toolchains/linux.cmake b/scripts/toolchains/linux.cmake
+index 68d90470..7508cd46 100644
+--- a/scripts/toolchains/linux.cmake
++++ b/scripts/toolchains/linux.cmake
+@@ -17,8 +17,8 @@ if(NOT _CMAKE_IN_TRY_COMPILE)
+     string(APPEND CMAKE_C_FLAGS_RELEASE_INIT " ${VCPKG_C_FLAGS_RELEASE} ")
+     string(APPEND CMAKE_CXX_FLAGS_RELEASE_INIT " ${VCPKG_CXX_FLAGS_RELEASE} ")
+ 
+-    string(APPEND CMAKE_C_FLAGS_INIT " ${CMAKE_C_FLAGS} ")
+-    string(APPEND CMAKE_CXX_FLAGS_INIT " ${CMAKE_CXX_FLAGS} ")
++    string(APPEND CMAKE_C_FLAGS_INIT " ${CMAKE_C_FLAGS} -fsanitize=$ENV{SANITIZER} ")

```

Comment:
Why -fsanitize is not enclosed by "if"? 
What will happen when $ENV{SANITIZER} is not set?

Reply:
To be honest, I don't really care. It's intended to work only in a very specific environment (OSS-Fuzz container) which guarantees for specific environment variables are set.